sos/plugins/memory.py

```diff
@@ -37,7 +37,7 @@ def setup(self):
         self.add_cmd_output("free", root_symlink="free")
         self.add_cmd_output([
             "free -m",
-            "swapon --bytes --show"
+            "swapon --summary --verbose"
```

Comment:
These two commands do not do the same thing:

```
# swapon --bytes --show
NAME      TYPE            SIZE       USED PRIO
/dev/dm-1 partition 4294963200 1771630592   -1
```

```
# swapon --verbose --summary
Filename				Type		Size	Used	Priority
/dev/dm-1                              	partition	4194300	1730108	-1
```

I'm also surprised that you've found something "too old" to have these options; they have been part of util-linux's swapon for a long time (RHEL5 is the only distribution I have to hand that still has an older version and that is not supported by today's upstream sos).

So I think we need to understand this better - there should be very few distributions using an older package and if there are then those can be handled, just for those distros, using the tagging class mechanism.